chcemvediet/apps/inforequests/tests/test_models/test_branch.py

```diff
@@ -484,17 +481,17 @@ def test_can_add_x_properties(self):
 
             # Check actions allowed when the last action deadline is not expired yet
             for can_add_property in can_add_properties:
-                value = getattr(branch, u'can_add_%s' % can_add_property)
+                value = getattr(branch, u'can_add_{}'.format(can_add_property))
                 expected = can_add_property in test.allowed
-                self.assertEqual(value, expected, u'can_add_%s is %s after %r' % (can_add_property, value, test.scenario))
+                self.assertEqual(value, expected, u'can_add_{} is {} after {}'.format(can_add_property, value, test.scenario))
```

Comment:
V podovnom format stringu sme mali pre niektore parametre format `%s` a pre niektore `%r`. To je rozdiel. Zrejme rovnako by sme to mali mat aj v novych format stringoch.

Reply:
Ano, ale `test.scenario` je `<type 'list'>`.  List nemaju metodu `__str__`. Potom pri volani `str()` sa vracia `repr(object)`.

Reply:
:ok: dakujem

Reply:
:ok: